engine/src/main/resources/templates/Inventory/validationErrorEmailBody.html

```diff
@@ -0,0 +1,40 @@
+{#include Inventory/insightsEmailBody}
+{#content-title}{action.context.event_name} Instant Notification - {action.timestamp.toStringFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-content__block">
+        <table class="rh-data-table rh-m-bordered">
+            <thead>
+            <tr>
+                <th>System</th>
+                <th>Error</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>
+                    {#if action.events[0].payload.host_id != null}<a href="https://console.redhat.com/insights/inventory/{action.events[0].payload.host_id}" target="_blank">{action.events[0].payload.display_name}</a>{#else}{action.events[0].payload.display_name}{/if}
```

Comment:
Now that we have the environment in the tests as well, could you use it here?

```suggestion
                    {#if action.events[0].payload.host_id != null}<a href="{inventory.url}/insights/inventory/{action.events[0].payload.host_id}" target="_blank">{action.events[0].payload.display_name}</a>{#else}{action.events[0].payload.display_name}{/if}
```